liblwgeom/lwin_wkt_parse.h

```diff
@@ -1,4 +1,4 @@
-/* A Bison parser, made by GNU Bison 3.4.  */
+/* A Bison parser, made by GNU Bison 3.4.2.  */
```

Comment:
Can we do something about not committing this file into repo? It looks completely generated, and seems to be going to be built anyway during build process.

Reply:
Unexpectedly I've been asking for this for a long time now. Contrary argument has always been: not everyone can easily get a parser generator nowadays ... Please please let's get rid of it now!

Reply:
I'd be ok with it. We just need to make sure it can also be generated with the tarball releases.